### PR TITLE
k830 support + support for NEW FN INVERSION

### PR DESCRIPTION
**Pull request by decibyte**

### CODE REVIEW

File: lib/logitech_receiver/descriptors.py [hunk 1]
```diff
@@ -191,6 +191,13 @@ def _D(name, codename=None, kind=None, wpid=None, protocol=None, registers=None,
 							_RS.hand_detection(),
 						],
 				)
+#'''
```
Comment (Lekensteyn) [inline lib/logitech_receiver/descriptors.py]: Can these markers be removed?

Comment (decibyte) [inline lib/logitech_receiver/descriptors.py]: Yes, of course. Sorry for not cleaning up after myself. I'll update the PR later today.

**Comment by decibyte** [comment]
@Lekensteyn I've now removed the stuff we talked about and also rewrapped the text in `devices.md`, which didn't comform to the 80col rule after I added k830 to the document.

**Comment by Lekensteyn** [comment]
Looks good, thanks!

**Pull request merged**